views/logs.ecr

```diff
@@ -9,7 +9,7 @@
     <% render "header" %>
     <main>
       <section id="livelog" class="card livelog">
-        <button id="download-logs" class="btn-primary"><a href="api/logs">Download logs</a></button>
+        <button id="download-logs" class="btn-primary"><a download href="api/logs">Download logs</a></button>
```

Comment:
Oh, didn't know about the `download` attribute, good find! https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/download